guides/using_hugging_face_integrations.md

```diff
@@ -112,6 +112,8 @@ The code above generates the following demo.
 
 As you can see, the demo looks the same, but it uses a webcam input instead of user-uploaded images.
 
+You can learn more about this feature, and how to use it with the new Blocks API in the [Treating Gradio apps like functions guide](/using_blocks_like_functions/)
```

Comment:
This guide already covers material similar to the proposed "Using apps like functions" guide so I debated whether or not we even need to add another guide.

I decided that we did because:
1. The material in this guide only covers Interfaces, which makes users think it only applies to Interfaces and demos built with Interfaces. 
2. It doesn't cover the full depth of the concept - just how you can change the input type. I think the concept that apps are functions that could be used anywhere is worth highlighting.